tests/commands/releases_finalize.rs

```diff
@@ -0,0 +1,80 @@
+use assert_cmd::Command;
+use mockito::{mock, Matcher};
+use predicates::str::contains;
+
+use crate::common;
+
+#[test]
+fn successfully_creates_a_release() {
+    let _server = mock(
+        "PUT",
+        "/api/0/projects/wat-org/wat-project/releases/wat-release/",
+    )
+    .match_body(Matcher::AllOf(vec![
+        Matcher::PartialJsonString(r#"{"projects":["wat-project"]}"#.to_string()),
+        Matcher::Regex("dateReleased".to_string()),
+    ]))
+    .with_status(200)
+    .with_header("content-type", "application/json")
+    .with_body(r#"{"dateReleased":"2020-06-29T12:16:49.368667Z","newGroups":0,"commitCount":0,"url":null,"data":{},"lastDeploy":null,"deployCount":0,"dateCreated":"2020-06-29T11:36:59.612687Z","lastEvent":null,"version":"wat-release","firstEvent":null,"lastCommit":null,"shortVersion":"wat-release","authors":[],"owner":null,"versionInfo":{"buildHash":null,"version":{"raw":"wat-release"},"description":"wat-release","package":null},"ref":null,"projects":[{"name":"test","platform":"javascript","slug":"test","platforms":["javascript"],"newGroups":0,"id":1861017}]}"#)
```

Comment:
you can put line breaks in raw strings. this is super hard to read.

Reply:
I'm aware of this, but this is a raw dump from the real request debug log. Some of them are quite large, and I didn't want it to obscure the rest of an actual test.